.circleci/config.yml

```diff
@@ -326,8 +326,8 @@ jobs:
       - run:
           name: Pull test image
           command: |
-            docker pull datasqrl/cmd-test-image:${CIRCLE_SHA1}
-            docker tag  datasqrl/cmd-test-image:${CIRCLE_SHA1} sqrl-test
+            docker pull datasqrl/cli-test-image:${CIRCLE_SHA1}
```

Comment:
Don't forget to create the new projects on docker hub

Reply:
I do not have any permission on DockerHub yet, can you help me out with that maybe? :)